extras/AFC.py

```diff
@@ -1002,5 +1002,20 @@ def afc_led (self, status, idx=None):
             self.gcode.respond_info( error_string)
         led.led_change(int(idx.split(':')[1]), status)
 
+    def TcmdAssign(self, CUR_LANE):
+        if CUR_LANE.map == 'NONE' :
+            for x in range(99):
+                cmd = 'T'+str(x)
+                if cmd not in self.tool_cmds:
+                    self.lanes[CUR_LANE.unit][CUR_LANE.name]['map'] = 'T'+str(x)
```

Comment:
Use the cmd variable that was created on line 1008
```suggestion
                    self.lanes[CUR_LANE.unit][CUR_LANE.name]['map'] = cmd
```

Reply:
Done